### DISCUSSION TITLE
How to rotate a PathGeometry/PathFigure used in an Image.Clip?

### DISCUSSION THREAD

**Discussion by Cfun1** [dotnet/maui#21468] [Q&A]
I have a PathGeometry used with in an Image.Clip, that i want to apply a rotation to it before applied to the image is ther ea way to achieve that ?
 i was expecting a Rotation property for PathFigure but it doesn't exist.

**Comment by symbiogenesis** [comment]
PathGeometry or PathFigures can be the Data for a Path object. And then the Path object has RotateTo()

**Reply by Cfun1** [reply]
> PathGeometry or PathFigures can be the Data for a Path object. And then the Path object has RotateTo()
I know but as described it can be the value of an `Image.Clip` also, in this case there is no `Path` object.
